File: Cenarius/WXProgressHUDModule.m
//
//  WXProgressHUDModule.m
//  Cenarius
//
//  Created by M on 2017/6/23.
//  Copyright © 2017年 M. All rights reserved.
//

#import "WXProgressHUDModule.h"

@implementation WXProgressHUDModule

@synthesize weexInstance;

WX_EXPORT_METHOD(@selector(show:))
WX_EXPORT_METHOD(@selector(dismiss:))

@end

File: Cenarius/WXProgressHUDModule.swift
//
//  WXProgressHUDModule.swift
//  Cenarius
//
//  Created by M on 2017/6/23.
//  Copyright © 2017年 M. All rights reserved.
//

import Foundation
import SVProgressHUD

public extension WXRouteModule {
    
    public func show() {
        SVProgressHUD.show()
    }
    
    public func dismiss() {
        SVProgressHUD.dismiss()
    }
}
